src/shrutiDJun20Assignments/PalindromeString1.java

```diff
@@ -0,0 +1,23 @@
+package shrutiDJun20Assignments;
+
+public class PalindromeString1 {
+
+	void palindromeStringCheck(String input) {
+		String rev = "";
+		for (int index = input.length()-1; index >= 0; index--) {
+			rev = rev + input.charAt(index);
+		}
+		if (input.equalsIgnoreCase(rev))
+			System.out.println("Given string '" + input + "' is palindrome.");
+		else
+			System.out.println("Given string '" + input
+					+ "' is not palindrome.");
+
+	}
+	public static void main(String[] args) {
+		PalindromeString1 palindromeString1 = new PalindromeString1();
+		palindromeString1.palindromeStringCheck("Radar");
```

Comment:
@ShrutiDJun20 @PalakSJun20 "Radar" is not a palindrome string. "radar" is . It is case sensitive

Reply:
changes done